Dockerfile

```diff
@@ -0,0 +1,123 @@
+# FROM mcr.microsoft.com/devcontainers/javascript-node:1-18
+FROM mcr.microsoft.com/devcontainers/javascript-node:22
```

Comment:
Any particular reason you chose this base image? Given this is a Python / Django project primarily, I would take a look at https://github.com/devcontainers/images/tree/main/src/python instead